tests/default.test.mjs

```diff
@@ -11,11 +11,13 @@ const appName = 'fancy-app-in-test';
 
 describe('basic functionality', function () {
   let tmpDir;
+  let emberCli = `ember-cli@5.11.0`;
```

Comment:
I don't like the test being explicit here, I want the version of ember-cli to be managed by the package.json and I set up a dependabot rule specifically to cover this

Reply:
ah, well it was using global before -- I'll investigate this separately